oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -247,7 +247,7 @@ public long getBytesWritten() {
 
     @Override
     public List<OSThread> getThreadDetails() {
-        return ProcessStat.getThreadIds(getProcessID()).stream().map(id -> new LinuxOSThread(getProcessID(), id))
+        return ProcessStat.getThreadIds(getProcessID()).stream().parallel().map(id -> new LinuxOSThread(getProcessID(), id))
```

Comment:
YES.  This is the primary use case envisioned by #2196.   the new thread call is heavy so this will help.   Not sure why we aren't sorting these, though...